ReverseString.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Scanner;
+
+public class ReverseString {
+    public static void main(String[] args) {
+
+        /************************************************
+        Write a Java program to reverse a string and 
+        Write a Java program to count the letters, spaces, 
+        numbers and other characters of an input string.
+        *************************************************/
+        
+        Scanner scan = new Scanner(System.in);
+        int escape = 1;
+        int letters = 0;
+        int numbers = 0;
+        int spaces = 0;
+        int othersChar = 0;
```

Comment:
rename these variables to escapeCount or escapeNumber to better descripe whats inside

Reply:
these below from letters to othersChar i mean, make enum from escape

Reply:
otherChars not othersChars